Errors Unchanged with arbitrary access tokens

I get this error with and without valid access tokens. Using the boto3 package in the command line I was able to download from the same s3 bucket using the same access tokens.

`[2018-08-08T22:54:11.903720Z] NIO [DEBUG] [s3transfer.tasks] Exception raised.
Traceback (most recent call last):
  File "/Users/sclay/nio/s3/lib/python3.6/site-packages/s3transfer/tasks.py", line 255, in _main
    self._submit(transfer_future=transfer_future, **kwargs)
  File "/Users/sclay/nio/s3/lib/python3.6/site-packages/s3transfer/download.py", line 353, in _submit
    **transfer_future.meta.call_args.extra_args
  File "/Users/sclay/nio/s3/lib/python3.6/site-packages/botocore/client.py", line 314, in _api_call
    return self._make_api_call(operation_name, kwargs)
  File "/Users/sc[2018-08-08T22:54:11.905772Z] NIO [DEBUG] [Custom Scheduler] Executing: <bound method Cache._remove_item of <nio.util.cache.Cache object at 0x107fa8d30>>
lay/nio/s3/lib/python3.6/site-packages/botocore/client.py", line 612, in _make_api_call
    raise error_class(parsed_response, operation_name)
botocore.exceptions.ClientError: An error occurred (400) when calling the HeadObject operation: Bad Request
[2018-08-08T22:54:11.906131Z] NIO [DEBUG] [s3transfer.utils] Releasing acquire 0/None
[2018-08-08T22:54:11.906365Z] NIO [ERROR] [sss.eaef55fd-8db1-4ced-a72a-29cde3d77d9f] File download failed
Traceback (most recent call last):
  File "/Users/sclay/nio/s3/s3/blocks/S3/amazon_s3_download_block.py", line 26, in process_signals
    self.file_name(signal).value)
  File "/Users/sclay/nio/s3/lib/python3.6/site-packages/boto3/s3/inject.py", line 172, in download_file
    extra_args=ExtraArgs, callback=Callback)
  File "/Users/sclay/nio/s3/lib/python3.6/site-packages/boto3/s3/transfer.py", line 307, in download_file
    future.result()
  File "/Users/sclay/nio/s3/lib/python3.6/site-packages/s3transfer/futures.py", line 73, in result
    return self._coordinator.result()
  File "/Users/sclay/nio/s3/lib/python3.6/site-packages/s3transfer/futures.py", line 233, in result
    raise self._exception
  File "/Users/sclay/nio/s3/lib/python3.6/site-packages/s3transfer/tasks.py", line 255, in _main
    self._submit(transfer_future=transfer_future, **kwargs)
  File "/Users/sclay/nio/s3/lib/python3.6/site-packages/s3transfer/download.py", line 353, in _submit
    **transfer_future.meta.call_args.extra_args
  File "/Users/sclay/nio/s3/lib/python3.6/site-packages/botocore/client.py", line 314, in _api_call
    return self._make_api_call(operation_name, kwargs)
  File "/Users/sclay/nio/s3/lib/python3.6/site-packages/botocore/client.py", line 612, in _make_api_call
    raise error_class(parsed_response, operation_name)
botocore.exceptions.ClientError: An error occurred (400) when calling the HeadObject operation: Bad Request`

--- a/amazon_s3_base_block.py
+++ b/amazon_s3_base_block.py
@@ -37,4 +37,4 @@ class S3Base(TerminatorBlock):
             's3',
             aws_access_key_id=self.creds().aws_access_key_id(),
             aws_secret_access_key=self.creds().aws_secret_access_key(),
-            aws_session_token=self.creds().aws_secret_access_key())
+            aws_session_token=self.creds().aws_session_token())

